### PR TITLE
[TRAFODION-533]Fixes for regressive SPJ calling

### PR DESCRIPTION
**Pull request by kevinxu021**

### CODE REVIEW

**Comment by selvaganesang** [comment]
Great. Can you please add the tests in the JIRA as part of the developer regressions

File: core/conn/jdbc_type2/native/SqlInterface.cpp [hunk 1]
```diff
@@ -2570,12 +2570,9 @@ SQLRETURN ALLOCSQLMXHDLS_SPJRS(SRVR_STMT_HDL *pSrvrStmt, SQLSTMT_ID *callpStmt,
 	if (pModule->module_name == NULL)
 	{
 		DEBUG_OUT(DEBUG_LEVEL_STMT,("***pModule->module_name == NULL  Call AllocStmtForRs()"));
-/* Commenting out for now - will be looked at when SPJ is supported
- #ifdef NSK_PLATFORM
 		CLI_AllocStmtForRS(callpStmt,
```
Comment (DaveBirdsall) [inline core/conn/jdbc_type2/native/SqlInterface.cpp]: Should we capture the return code here? retcode = CLI_AllocStmtForRS...

Comment (xwq) [inline core/conn/jdbc_type2/native/SqlInterface.cpp]: @DaveBirdsall Actually, CLI_AllocStmtForRS is a macro. retcode will capture the return code of "SQL_EXEC_AllocStmtForRS" (Debug.h +125). So we don't need capture the return code here.

Comment (kevinxu021) [inline core/conn/jdbc_type2/native/SqlInterface.cpp]: @selvaganesang For spj tests, a spj jar package is required while create a library. I'm not sure the path of the JAR. it seems to be a little complicated. The JAR should be existing on server-side(every node). i'm not sure the IP address/host name(where to upload) while the tests running.

**Comment by DaveBirdsall** [comment]
I am happy to commit this change. I assume that adding tests can be done as a separate pull request. Is this OK with you, @selvaganesang?

**Comment by selvaganesang** [comment]
Fine with me. The SPJ tests can be modeled after the regression tests available at sql/regress/udr/TEST*. It is possible to add to an existing test suite. Adding a test would help to detect this issue earlier as part of developer regressions going forward if it gets broken for some reason.

**Pull request merged**